internal/app/admin/http/cache.go

```diff
@@ -106,6 +106,38 @@ func edsDumpHandler(o *orchestrator.Orchestrator) http.HandlerFunc {
 	}
 }
 
+func versionHandler(o *orchestrator.Orchestrator) http.HandlerFunc {
+	return func(w http.ResponseWriter, req *http.Request) {
+		cdsKey := filepath.Base(req.URL.Path)
+		if cdsKey == "" {
+			w.WriteHeader(http.StatusBadRequest)
+			s, _ := stringify.InterfaceToString(&marshallable.Error{
+				Message: "Empty key",
+			})
+			_, _ = w.Write([]byte(s))
+		}
+
+		c := orchestrator.Orchestrator.GetReadOnlyCache(*o)
+		resp, err := c.FetchReadOnly(cdsKey)
+		if err != nil {
+			w.WriteHeader(http.StatusNotFound)
+			s, _ := stringify.InterfaceToString(&marshallable.Error{
+				Message: err.Error(),
+			})
+			_, _ = w.Write([]byte(s))
+			return
+		}
+
+		version := resp.Resp.GetPayloadVersion()
+		_, e := w.Write([]byte(version))
```

Comment:
do we want to wrap the response in an object? We could choose to add more fields later without breaking the clients.